### ISSUE TITLE
Funding for Open Access publication

### ISSUE THREAD

**Issue by stephwright** [mozillascience/openguides#1]
Recommendations from this thread: https://twitter.com/shefw/status/951195568692379648
* ask publisher for (partial) fee wavier
* upload a preprint (in any case!)
* consider an alternative less expensive OA journal e.g. PeerJ or Royal Society Open Science.
  -  Latter APC is just £900/€1080/$1260 http://rsos.royalsocietypublishing.org/page/charges
* Resource by @protohedgehog  https://figshare.com/collections/How_to_make_your_work_100_Open_Access_for_free_and_legally_multi-lingual_/3943972

**Comment by gedankenstuecke** [comment]
I feel that the `just £900/€1080/$1260` should have scare quotes around the *"just"*, as that's still prohibitively expensive for many.

**Comment by stephwright** [comment]
Indeed it's higher than the student who reached out to me could cover.

**Comment by gedankenstuecke** [comment]
An item to add for this problem: 
* Many university libraries have open access funds that can be accessed to help to offset APCs. Depending on the uni these can be open to students 

(e.g. I got a 50% APC cover for my Sci-Hub article last year from the library of the University of Frankfurt)

**Comment by safiume** [comment]
Also crowdfunding websites like https://generosity.com and gofundme, pateron, or even paypal and the like can help.

**Comment by Kevin-Mattheus-Moerman** [comment]
Yep universities often cover some of the APC. In the case of MIT up to 1000 dollar is covered if the journal is listed in the [directory of open access journals](https://doaj.org/). BTW I started building https://openaccessclinic.github.io/OA_clinic I could use help. We could add info developed here on that site too.

**Comment by stephwright** [comment]
Thanks @safiume ! I've been having more and more interaction with Patreon of late for my personal support of open projects.  So sad that we have to go that route, sometimes.

Now @Kevin-Mattheus-Moerman ... how is it I didn't know you had that guide!?!  Depending on how much input we get, I was thinking of the possibility of making this guide a GitBook (so folks could easily navigate to pieces they were interested in).  Your guide is a great starting point!

**Comment by gedankenstuecke** [comment]
Just a minor comment ad Patreon: It can be really useful, but depending on your jurisdiction it can be a pain to set up in a way that you don't run into tax troubles along the way. At least my experience with openSNP was that it was less than straight forward to set this up in Germany.

**Comment by stephwright** [comment]
Good to know... thanks @gedankenstuecke .  I think we're better off providing lists of options rather than trying to do any sort of one-size-fits-all anyway.

**Comment by gedankenstuecke** [comment]
👍 agreed on that, would be nice if the options provide pros and cons, that's why I thought about bringing it up 😄

**Comment by ababaian** [comment]
Although case-by-case; asking your particular department is an option. We're lucky in that 3 students are from the same department/faculty which means we can request a little bit of support which is distinct from the money from the University library. We'll see  how that goes.

**Comment by ababaian** [comment]
Thanks for all the suggestions. After filling in the paperwork and getting a letter from the department head, BMC has waived all costs for publication (if the paper is accepted).

tbh wasn't expecting that at all but it's very nice of them. They wrote that it's not guaranteed that they will do this and it's decided on a case-by-case basis. For reference the original fee was $3000 CAD, the university library covers 15% of that and the hackathon we started at was willing to cover another $500 which left ~$2000 CAD which was required.

**Comment by stephwright** [comment]
This is such great news, @ababaian !!!!  And look what your question started.  I will work on creating this into a guide though will probably have to wait until February... unless someone else wants to take it up.  If we start issues for other ideas for guides, I may enter this as a project for our upcoming Global Sprint, May 10-11.
